api/client/lib.py

```diff
@@ -598,14 +598,19 @@ def get_geo_centre(access_token, api_host, region_id):
 
 
 @memoize(maxsize=None)
```

Comment:
Need to move the memoize decorator down to the integer `get_geojson()` one

Memoization doesn't work with functions that take a list as their input.